### PR TITLE
FEAT: Add Avatar Component

### PR DESCRIPTION
**Pull request by VishalPatil18**
### 📝 Description

Added the Avatar Component. It is used for profile images or user profile menus.

### 🚀 New behavior

1. Added the two variants
   - text
   - image
2. The component accepts props like
   - `variant` - The variant of the avatar to use. e.g. image, text
   - `source` - The source of the image to be used in avatar
   - `text` - For the text variant the text to be displayed in the avatar
   - `shape` - Shape of Avatar. e.g. Circle, square, rounded, etc
   - `background` - For the text variant the background color to be used
   - `color` - For the text variant the font color to be used
   - `size` - The size of the avatar component. e.g. xs, sm, md, lg, xl

> All props are optional

### 🖼 Screenshots

Screenshot 2022-03-03 at 19 06 57

### 💣 Is this a breaking change?
- [ ] Yes
- [x] No

### 🔬 Is the code self-reviewed?
- [x] Yes
- [ ] No

### 📋 Is the code well-formatted?
- [x] Yes
- [ ] No

### 🔗 Related to Issue #4 

> NOTE: The changes in `App.jsx` file are just for the development purpose of the current component.

### CODE REVIEW

File: src/components/avatar/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          className={`avatar av-${shape} av-${size}`}
+          src={source}
+          alt="avatar"
+        />
+      ) : (
+        <span
+          className={`avatar av-text av-${shape} av-${size}`}
+          style={{ backgroundColor: background, color: color }}
+        >
+          {text.substring(0, 2)}
```
Comment (Ayushi20-19) [inline src/components/avatar/Avatar.jsx:46]: can u explain me this line, like if the text is ayushi it will return ay?

**Review by Ayushi20-19 — COMMENTED** [review]
code seems nice to me great work around variations...

**Review by Ayushi20-19 — COMMENTED** [review]
code seems nice to me great work around variations...

File: src/components/avatar/Avatar.jsx (hunk 1 above)
Comment (Ayushi20-19) [reply src/components/avatar/Avatar.jsx:46]: got it, it is working fine

**Review by kart-c — APPROVED** [review]
Looks good VIshal 🚀

Like how you have explained all the changes in the PR description

Also like the fallback provided if the user does not specify one of the props 👏

**Pull request merged**